snapcraft/cli/store.py

```diff
@@ -252,18 +278,91 @@ def list_registered():
     snapcraft.list_registered()
 
 
+@storecli.command('export-login')
+@click.argument('login_file', metavar='FILE', type=click.File('w'))
+@click.option('--packages', metavar='<packages>',
+              help='Comma-separated list of packages to limit token access')
+@click.option('--channels', metavar='<channels>',
+              help='Comma-separated list of channels to limit token access')
+@click.option('--acls', metavar='<acls>',
+              help='Comma-separated list of ACLs to limit token access. Valid '
+                   "ACLs are 'package_upload', 'package_access', and "
+                   "'package_manage'")
+def export_login(login_file: TextIO, packages: str, channels: str, acls: str):
+    """Save attenuated login configuration for a store account in FILE.
+
+    This file can then be used to login to the given account with the
```

Comment:
The command is `login`, but the English verb is “log in”. You could use quotes to clarify that you’re using the command name here.

Reply:
Ah, I'm not even really talking about the command here, more the action indeed. I'll use "log in".